src/api/app/helpers/webui/maintenance_incident_helper.rb

```diff
@@ -111,7 +111,7 @@ def outgoing_request_links(requests)
             tag.i(nil, class: "fas fa-flag pr-1 request-flag-#{request['state']}", title: "Release request in state '#{request['state']}'")
           end,
           # rubocop:disable Rails/OutputSafety
-          "Created #{fuzzy_time(request.created_at)}".html_safe
+          FuzzyTimeComponent.new(time: request.created_at).human_time_ago.html_safe
```

Comment:
I think we can safety change it for `sanitize` as described in the documentation https://apidock.com/rails/v6.1.3.1/String/html_safe. Even though the field `created_at` is never under user control.